pkgs/development/python-modules/sqlalchemy-i18n/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, lib, fetchPypi, buildPythonPackage
```

Comment:
The mix in styles (one per line vs long lines) irks me a bit here :smile:

Reply:
I think I started doing this, everything but python packages are included in the first line, python packages are then entered 1 per line.

Generally `stdenv, lib, fetchPypi, buildPythonPackage` are unlikely to ever change, so just have them together.

I guess it's just a personal preference

Reply:
I ran nixpkgs-fmt on the file :)